pre-index.php

```diff
@@ -602,9 +602,15 @@ class="onoffswitch-checkbox"/>
     </nav>
     <nav id="stats">
         <div class="switch-container">
-            <!--<div class="switch-container">
-                <div><center><a href="stats">Full Stats</a></center></div>
-            </div>-->
+            <?php 
+            if ($worldopoleUrl != "") {
+                ?>
+                <div class="switch-container">
+                    <div><center><a href="<?= $worldopoleUrl ?>">Full Stats</a></center></div>
```

Comment:
I feel just the blue hyperlink `Full Stats` is a bit uninviting and easily overlooked, perhaps when enabled could trigger the Worldopole Icon along with it? (Or icon of choice)
Using on my map, no issues.